CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.1)
```

Comment:
Which function requires version 3.1?

Reply:
We are using version 3.1 as a minimum in `synfig-studio/CMakeLists.txt` and `synfig-core/CMakeLists.txt`.

Reply:
Yeah, they need to be fixed too. Added issue here: https://github.com/synfig/synfig/issues/1509